adagrams/game.py

```diff
@@ -1,11 +1,132 @@
+import random
+
+def dictionary_to_list(dict):
+    letter_list = []
+    for key, value in dict.items():
+        for i in range(value): 
+            letter_list.append(key)
+    
+    return letter_list
+
 def draw_letters():
-    pass
+    letter_pool = {
+        "A": 9,
+        "B": 2,
+        "C": 2,
+        "D": 4,
+        "E": 12,
+        "F": 2,
+        "G": 3,
+        "H": 2,
+        "I": 9,
+        "J": 1,
+        "K": 1,
+        "L": 4,
+        "M": 2,
+        "N": 6,
+        "O": 8,
+        "P": 2,
+        "Q": 1,
+        "R": 6,
+        "S": 4,
+        "T": 6,
+        "U": 4,
+        "V": 2,
+        "W": 2,
+        "X": 1,
+        "Y": 2,
+        "Z": 1,
+    }
+
+    possible_letters = dictionary_to_list(letter_pool)
+
+    player_hand = random.sample(possible_letters, 10)
+    return player_hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word = word.upper()
+
+    letter_dict = dict()
+
+    for letter in letter_bank:
+        if letter in letter_dict:              
+            letter_dict[letter] +=1
+        else:
+            letter_dict[letter] = 1
```

Comment:
Great use of a frequency map! 

This is more of a Python-y nitpick but there should always be a space between operators and values. 

```suggestion
    for letter in letter_bank:
        if letter in letter_dict:              
            letter_dict[letter] += 1
        else:
            letter_dict[letter] = 1
```